e2e/e2etests/test_bitcoin_donation.go

```diff
@@ -0,0 +1,44 @@
+package e2etests
+
+import (
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/pkg/constant"
+	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
+	zetabitcoin "github.com/zeta-chain/node/zetaclient/chains/bitcoin"
+)
+
+func TestBitcoinDonation(r *runner.E2ERunner, args []string) {
+	// ARRANGE
+	// Given BTC address
+	r.SetBtcAddress(r.Name, false)
+
+	// Given "Live" BTC network
+	stop := r.MineBlocksIfLocalBitcoin()
+	defer stop()
+
+	// Given amount to send
+	require.Len(r, args, 1)
+	amount := parseFloat(r, args[0])
+	amountTotal := amount + zetabitcoin.DefaultDepositorFee
+
+	// Given a list of UTXOs
+	utxos, err := r.ListDeployerUTXOs()
+	require.NoError(r, err)
+	require.NotEmpty(r, utxos)
+
+	// ACT
+	// Send BTC to TSS address with donation message
+	memo := []byte(constant.DonationMessage)
+	txHash, err := r.SendToTSSFromDeployerWithMemo(amountTotal, utxos, memo)
+	require.NoError(r, err)
+
+	// ASSERT after 20 seconds
+	time.Sleep(time.Second * 20)
```

Comment:
The donation tx currently triggers nothing. Zetaclient only print a log on donation. This is a indirectly way to make sure nothing has happened. We have a group of Bitcoin E2E tests to run, what I can do is to move this donation as the 1st test so to avoid wasting time. wdyt?

Reply:
I moved as the 1st test in Bitcoin group in commit: https://github.com/zeta-chain/node/pull/3025/commits/b2c149c4124f27c83f2ef6f4a55f76f8c3534433